### PR TITLE
build: change structure to ts

### PR DESCRIPTION
**Pull request by erhise**
EDIT:
Remove all unused code for now.
Created examples folder (temporary) just to be able to run something

EDIT2:
Made some changes to resolve path logic
Right now it works with
```
$ npm run global // to install it globally
$ cd custom-project && offline-blitz filename.ts
```

EDIT3:
I removed adding file extension => making fileName method kind of redundant

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -13,7 +13,7 @@ const tsCompiler = register({
 
 blitz(path, tsCompiler);
 
-function blitz(filePath, compiler) {
+function blitz(filePath: any, compiler: any) {
```
Comment (siguse) [inline src/index.ts:16]: Why any here? filePath: string, compiler: Service should work I think

File: src/index.ts [hunk 2]
```diff
@@ -45,23 +45,25 @@ function compile(filePath, { compile }) {
   return null;
 }
 
-function newSandbox(code) {
-  const sandbox = fork('eval.js');
+function newSandbox(code: any) {
```
Comment (siguse) [inline src/index.ts:48]: code: string :)

File: src/index.ts [hunk 3]
```diff
@@ -45,23 +45,25 @@ function compile(filePath, { compile }) {
...
-  sandbox.send(code);
+  if (sandbox.send !== undefined) {
+    sandbox.send(code);
+  }
   return sandbox;
 }
 
-function onExecutionError(sandbox) {
-  return (executionError) => {
+function onExecutionError(sandbox: any) {
```
Comment (siguse) [inline src/index.ts:58]: sandbox: ChildProcess :)

File: src/index.ts [hunk 4]
```diff
@@ -45,23 +45,25 @@ function compile(filePath, { compile }) {
...
+  return (executionError: Error) => {
     console.log('Execution error:\n', executionError);
     sandbox.kill();
   };
 }
 
-function onSandboxAborted(sandbox) {
-  return (exitCode) => {
+function onSandboxAborted(sandbox: any) {
+  return (exitCode: any) => {
```
Comment (siguse) [inline src/index.ts:66]: exitCode: number

File: src/index.ts [hunk 5]
```diff
@@ -25,17 +25,17 @@ function blitz(filePath, compiler) {
...
 }
 
-function runSandbox(filePath, compiler) {
+function runSandbox(filePath: string, compiler: any) {
   const code = compile(filePath, compiler);
   return code !== null ? newSandbox(code) : null;
 }
 
-function compile(filePath, { compile }) {
+function compile(filePath: string, { compile }: any) {
```
Comment (siguse) [inline src/index.ts:38]: maybe we should just take the compiler as an argument here (compiler: Service) and call compiler.compile?

File: src/app.ts [hunk 6]
```diff
@@ -0,0 +1 @@
+console.log('hej');
```
Comment (siguse) [inline src/app.ts:1]: Should this be included? 🤔

**Comment by erhise** [comment]
made some cleanup @siguse

**Review by siguse — APPROVED** [review]

**Pull request merged**